package.json

```diff
@@ -10,7 +10,7 @@
   ],
   "scripts": {
     "lint": "esw webpack.config.* src tools test --color",
-    "clean-dist": "rm -rf ./lib && mkdir lib",
+    "clean-dist": "rimraf -rf ./lib && mkdir lib",
```

Comment:
--rf is not required when using rimraf :)

Reply:
@Lukasz-pluszczewski 

done